src/libos/src/process/do_vfork.rs

```diff
@@ -201,3 +228,62 @@ pub fn handle_force_stop() {
         }
     }
 }
+
+// Wait4 unwaited child which are created with vfork and directly exit without calling execve.
+pub fn wait4_exit_child_created_with_vfork(
+    parent_pid: pid_t,
+    child_filter: &ProcessFilter,
+) -> Result<(pid_t, i32)> {
```

Comment:
I think it makes more sense for the return type of `wait4_exit_child_created_with_vfork` to be `Option<(pid_t, i32)>`,  the error of no child process is handled in `do_wait4`.

Reply:
Done. Thanks for your review.